lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -658,8 +658,8 @@ void TypeAnalysis::visitOperation(Operation *op,
           AtenZero_Op, AtenIndexTensorOp, ValsemVariantAtenIndexPutImplOp,
           AtenIndexPutOp, ValsemVariantAtenCopyOp, AtenZeroOp,
           AtenIndexPutHackedTwinOp, AtenMaskedFillScalarOp, AtenFlipOp,
-          PrimAbsScalarOp, AtenNumpyTOp, AtenTriuOp, AtenMaskedFillTensorOp>(
-          op)) {
+          PrimAbsScalarOp, AtenNumpyTOp, AtenTriuOp, AtenMaskedFillTensorOp,
+          AtenDivIntOp>(op)) {
```

Comment:
I would have expected `AtenDivIntOp` to not require any `RefineTypes` handling since it always takes scalar ints as inputs and produces scalar floats as outputs. Is this change needed?